etc/profile-a-l/authenticator-rs.profile

```diff
@@ -51,5 +51,7 @@ private-etc alternatives,ca-certificates,crypto-policies,dconf,fonts,gtk-2.0,gtk
 private-tmp
 
 dbus-user filter
+dbus-user.own uk.co.grumlimited.authenticator-rs
```

Comment:
It doesn't work without this, I've rechecked again and dbus shows a string communicating.